src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java

```diff
@@ -72,6 +72,27 @@ public void childThreadTest() {
         assert(childThread.ftzValue == false);
     }
 
+    @Test(expectedExceptions = {NullPointerException.class})
+    public void nullfileNameTest() {
```

Comment:
Just a suggestion - Perhaps suffix the test name with `TestResource` as `nullfileNameTestResource` to name it after source method being tested here `pathToTestResource` rather than generic `Test` suffix from maintainability standpoint. Same applies to other instances too.